Package.swift

```diff
@@ -17,8 +20,8 @@ let package = Package(
     targets: [
         .binaryTarget(
             name: "FingerprintPro",
-            url: "https://fpjs-public.s3.amazonaws.com/ios/2.1.4/FingerprintPro-2.1.4-a1970ecc0ce58c1bfcc30968c0a97009f7b3731b2a652c13e4d6d989b32573a3.xcframework.zip",
-            checksum: "a1970ecc0ce58c1bfcc30968c0a97009f7b3731b2a652c13e4d6d989b32573a3"
+            url: "https://fpjs-public.s3.amazonaws.com/ios/\(version)/FingerprintPro-\(version)-\(checksum).xcframework.zip",
+            checksum: "\(checksum)"
```

Comment:
`checksum` is already a string, so no need for string interpolation here.
```suggestion
            checksum: checksum
```